website/views/static/faq.jade

```diff
@@ -0,0 +1,115 @@
+extends ./layout
+
+block vars
+  - var layoutEnv = env
+
+block title
+  title Habitica &VerticalLine;&nbsp;
+    =env.t('FAQ')
+
+block content
+  script(type='text/javascript').
```

Comment:
Moved to an Angular controller in `static.js`, where we appear to have several similar oddments.